dbt/include/teradata/macros/materializations/incremental/helpers.sql

```diff
@@ -1,21 +1,47 @@
-{% macro incremental_upsert(on_schema_change, tmp_relation, target_relation, existing_relation, unique_key=none, statement_name="main") %}
-    {% set dest_columns = process_schema_changes(on_schema_change, tmp_relation, existing_relation) %}
-    {% if not dest_columns %}
-        {%- set dest_columns = adapter.get_columns_in_relation(target_relation) -%}
-    {% endif %}
-    {%- set dest_cols_csv = dest_columns | map(attribute='quoted') | join(', ') -%}
-
-    {%- if unique_key is not none -%}
-    DELETE
-    FROM {{ target_relation }}
-    WHERE ({{ unique_key }}) IN (
-        SELECT ({{ unique_key }})
-        FROM {{ tmp_relation }}
-    );
-    {%- endif %}
-
-    INSERT INTO {{ target_relation }} ({{ dest_cols_csv }})
-       SELECT {{ dest_cols_csv }}
-       FROM {{ tmp_relation }}
-    ;
-{%- endmacro %}
+{% macro teradata__get_delete_insert_merge_sql(target_relation, tmp_relation, unique_key, dest_columns) %}

```

Comment:
can we move this macro to [strategies.sql](https://github.com/Teradata/dbt-teradata/pull/66/files?diff=split&w=1#diff-69d4bba51fa7ec1d765345cfec0250eda4741faa4a5c619e7dc716770ed4e2c8) as other strategy was defined in this file

Reply:
Yes sure, that will better option.